bloom/generators/debian/generator.py

```diff
@@ -471,6 +485,10 @@ def __process_template_folder(path, subs):
             os.path.relpath(item),
             os.path.relpath(template_path)))
         result = em.expand(template, **subs)
+        # Skip the result is empty
+        if len(result) == 0 and \
+           os.path.basename(template_path) in ['copyright']:
+            continue
```

Comment:
Before continuing here we need to mark the template file as processed so it will not b copied verbatim to the processed directory. Adding the line `processed_items.append(item)` should do the trick.